device/sycl/src/seeding/seed_finding.sycl

```diff
@@ -5,6 +5,9 @@
  * Mozilla Public License Version 2.0
  */
 
+// System include(s).
+#include <algorithm>
+
```

Comment:
It's not harmful, but I guess this was left in by mistake, no?

Reply:
I intentionally added it because of an std::transform used in this file. The std::transform was already there, not added in this PR, but seemed better to explicitly include it

Reply:
:+1: